plugins/woocommerce/client/blocks/assets/js/blocks/category-title/edit.tsx

```
@@ -0,0 +1,212 @@
+				tagName={ TagName }
+				placeholder={ __( 'No title', 'woocommerce' ) }
+				value={ rawTitle }
+				onChange={ ( v ) => setTitle( v ) }
+				__experimentalVersion={ 2 }
+				{ ...blockProps }
+			/>
+		) : (
+			<ContainerElement
+				tagName={ TagName }
+				{ ...blockProps }
+				dangerouslySetInnerHTML={ {
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)